go.mod

```diff
@@ -3,21 +3,54 @@ module github.com/crossplane/crossplane
 go 1.13
 
 require (
+	cloud.google.com/go/firestore v1.1.1 // indirect
+	cloud.google.com/go/spanner v1.3.0 // indirect
+	github.com/Azure/azure-sdk-for-go v40.4.0+incompatible // indirect
+	github.com/Azure/go-autorest/autorest/to v0.3.0 // indirect
+	github.com/Shopify/logrus-bugsnag v0.0.0-20171204204709-577dee27f20d // indirect
+	github.com/apex/log v1.1.2 // indirect
+	github.com/bitly/go-simplejson v0.5.0 // indirect
+	github.com/bshuster-repo/logrus-logstash-hook v0.4.1 // indirect
+	github.com/bugsnag/bugsnag-go v1.5.3 // indirect
+	github.com/bugsnag/panicwrap v1.2.0 // indirect
 	github.com/crossplane/crossplane-runtime v0.5.1-0.20200303183616-3d4c771a3062
 	github.com/crossplane/crossplane-tools v0.0.0-20200219001116-bb8b2ce46330
+	github.com/denverdino/aliyungo v0.0.0-20200313034858-035c7b930448 // indirect
+	github.com/dnaeon/go-vcr v1.0.1 // indirect
 	github.com/docker/distribution v2.7.1+incompatible
+	github.com/docker/go-metrics v0.0.1 // indirect
+	github.com/docker/libtrust v0.0.0-20160708172513-aabc10ec26b7 // indirect
+	github.com/garyburd/redigo v1.6.0 // indirect
 	github.com/ghodss/yaml v1.0.0
-	github.com/google/go-cmp v0.3.1
+	github.com/go-sql-driver/mysql v1.5.0 // indirect
+	github.com/gofrs/uuid v3.2.0+incompatible // indirect
+	github.com/google/go-cmp v0.4.0
+	github.com/gorilla/handlers v1.4.2 // indirect
+	github.com/gorilla/mux v1.7.4 // indirect
+	github.com/gucumber/gucumber v0.0.0-20180127021336-7d5c79e832a2 // indirect
+	github.com/jessevdk/go-flags v1.4.0 // indirect
+	github.com/kardianos/osext v0.0.0-20190222173326-2bc1f35cddc0 // indirect
+	github.com/ncw/swift v1.0.50 // indirect
 	github.com/onsi/gomega v1.7.0
 	github.com/opencontainers/go-digest v1.0.0-rc1 // indirect
+	github.com/opencontainers/image-spec v1.0.1 // indirect
 	github.com/pkg/errors v0.8.1
-	github.com/prometheus/client_golang v1.1.0 // indirect
+	github.com/pkg/sftp v1.11.0 // indirect
+	github.com/rs/zerolog v1.18.0 // indirect
+	github.com/satori/go.uuid v1.2.0 // indirect
+	github.com/shiena/ansicolor v0.0.0-20151119151921-a422bbe96644 // indirect
 	github.com/spf13/afero v1.2.2
+	github.com/yvasiyarov/go-metrics v0.0.0-20150112132944-c25f46c4b940 // indirect
+	github.com/yvasiyarov/gorelic v0.0.7 // indirect
+	github.com/yvasiyarov/newrelic_platform_go v0.0.0-20160601141957-9c099fbc30e9 // indirect
+	go.pedge.io/lion v0.0.0-20190619200210-304b2f426641 // indirect
 	gopkg.in/alecthomas/kingpin.v2 v2.2.6
+	gopkg.in/inconshreveable/log15.v2 v2.0.0-20200109203555-b30bc20e4fd1 // indirect
 	k8s.io/api v0.17.3
 	k8s.io/apiextensions-apiserver v0.0.0-20190918161926-8f644eb6e783
 	k8s.io/apimachinery v0.17.3
 	k8s.io/client-go v0.17.3
+	rsc.io/letsencrypt v0.0.3 // indirect
```

Comment:
It seems this is currently failing due to `make check-diff`; likely because `go mod tidy` would remove some of these indirect dependencies.

Reply:
I am fixing this now.